src/api/bind-contract-function.js

```diff
@@ -19,7 +19,7 @@ function bindContractFunction(functionAbi) {
         }
         if (!isFunction(params[params.length - 1])) return console.error("Callback required");
         var callback = params.pop();
-        return ethrpc.callContractFunction(payload, function (err, response) {
+        return self.rpc.callContractFunction(payload, function (err, response) {
```

Comment:
flash scripts fail at this line. ` return self.callContractFunction(payload, function (err, response) {...`

TypeError: self.callContractFunction is not a function